app/decorators/spree/easypost/shipment_decorator.rb

```diff
@@ -0,0 +1,11 @@
+module EasyPost
+  module ShipmentDecorator
+    EXCLUDED_RATES = %w[ ParcelSelect ]
+
+    def allowed_rates
+      self.rates.reject { |r| EXCLUDED_RATES.include?(r["service"]) }
+    end
+
+    ::EasyPost::Shipment.prepend(self)
```

Comment:
Out of curiosity, is it common practice to inject custom logic into 3rd party objects when we could just have a method that takes a shipment and returns available rates in our own namespace code?